.github/workflows/build_and_test.yml

```diff
@@ -75,6 +75,23 @@ jobs:
       - name: run type checking
         run: uv run mypy
 
+  lint_html_formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Read .nvmrc
+        id: nvmrc
+        run: echo "nodeVersion=$(cat .nvmrc)" >> "$GITHUB_OUTPUT"
+      - name: Setup node ${{ steps.nvmrc.outputs.nodeVersion }}
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4
+        with:
+          node-version: ${{ steps.nvmrc.outputs.nodeVersion }}
+      - name: install dependencies
+        run: npm ci
+      - name: check HTML formatting
+        run: make format-html
```

Comment:
I think `--write` here will still return 0 if it comes across things it can fix (after dumping the files into an ephemeral void), we probably just want to run `--check`

I can double check the behviour as I've forgotten

Reply:
yeah I think you're right - trying to work out how to get a 1 now 👀

Reply:
Fixed in 36911594cd40c8ac7c17b9d4d63210c3531defd2